week-2-practise/level-2/6-map.js

```diff
@@ -4,8 +4,11 @@
 //    - each item in `percentages` formatted by your function
 
 var percentages = [1, 23, 92, 18];
+function percentagesFormatted(numbers) {
+  return numbers + "%";
+}
```

Comment:
Be careful with naming. This function formats a single number as a percentage, so `percentagesFormatted` and `numbers` aren't good names for the function or the parameter it takes.